osu.Game.Rulesets.Mania/UI/Column.cs

```diff
@@ -187,10 +185,17 @@ public Color4 AccentColour
             }
         }
 
-        public void Add(DrawableHitObject<ManiaHitObject, ManiaJudgement> hitObject)
+        public double TimeSpan

```

Comment:
No, it's the length of time which the coordinates of the container spans. Saying it that way I guess length might also be okay but PreEmpt is more descriptive imo.